CMakeLists.txt

```diff
@@ -15,6 +15,17 @@ else()
   set(BUILD_DEPS "/usr/local/osquery")
 endif()
 
+if(WIN32)
+  set(WINDOWS_DEP_DIR "$ENV{ChocolateyInstall}/lib")
+  string(REGEX REPLACE "\\\\" "/" WINDOWS_DEP_DIR ${WINDOWS_DEP_DIR})
+  list(APPEND WINDOWS_DEP_LIST ${BUILD_DEPS})
+  list(APPEND WINDOWS_DEP_LIST ${WINDOWS_DEP_DIR})
+  list(APPEND WINDOWS_DEP_LIST "$ENV{HOMEDRIVE}/local/boost-msvc14")
```

Comment:
Can we add a line to the provision script to move it into `${WINDOWS_DEP_DIR}` after the install?